src/db/seeders/20230422183204-test.ts

```diff
@@ -1,20 +1,37 @@
+// @ts-nocheck
 'use strict';
 import data from "../seedgen/mock-data-gen"
```

Comment:
`20230422183204-test.js` also compiles. I do not experience module import problems on my local setup. And you also may use the following:
```
// eslint-disable-next-line @typescript-eslint/no-var-requires
const data = require('../seedgen/mock-data-gen');
```